puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release_test.yaml

```diff
@@ -30,7 +30,7 @@
           - trusty
           - xenial
     execution-strategy:
-      combination-filter: 'os == "el7" || os == "f24" || os == "jessie" || (os == "stretch" && major_version != "1.14" && major_version != "1.15") || (os == "trusty" && major_version == "1.15") || os == "xenial"'
+      combination-filter: 'os == "el7" || os == "jessie" || (os == "stretch" && major_version != "1.15" && major_version != "1.16") || (os == "trusty" && major_version == "1.16") || os == "xenial"'
```

Comment:
Also, I thought stretch support was in 1.16... is that meant to be != 1.14 / 1.15?